### PR TITLE
Update ctf_map READ.ME screenshot instructions

### PR DESCRIPTION
**Pull request by a-blob**
Add more details to the instructions about taking a screenshot to make the process easier and ensure screenshots with better quality. 
`` are also added to the block string names to make them more visually appealing.

### CODE REVIEW

**Comment by LuismiInTheZone** [comment]
> Add more details to the instructions about taking a screenshot to make the process easier and ensure screenshots with better quality. `` are also added to the block string names to make them more visually appealing.

Not every computer has a F12 for a screenshot, for example mine does nothing if I press F12, and with my last computer happened the same. For example, old and new computers, have got a special key that puts ImpPt, that means ''Imprimir Pantalla'' in spanish

File: mods/ctf/ctf_map/README.md [hunk 1]
```diff
@@ -119,5 +119,12 @@ Areas where chests can be placed. Some maps allow placing chests anywhere, while
...
-
-It should be named `screenshot.png`.
+You can take a screenshot easily by doing the following:
+1. Press F1 on your keyboard to hide the HUD
+2. Press F2 on your keyboard to hide the chat log
+3. *Optional:* Press R on your keyboard to enable unlimited viewing range if part of the map is not rendered
+4. Try to find a good view that shows most of the map
+5. Press F12 on your keyboard to take a screenshot
+6. You can find your screenshot inside `\[Minetest folder]/screenshots`
+
```
Comment (farooqkz) [inline mods/ctf/ctf_map/README.md:129]:
```suggestion
1. Hide the HUD. By default <kbd>F1</kbd> does that.
2. Hide the chat log. By default <kbd>F2</kbd> does that.
3. *(Optional)* Enable unlimited view range if all important parts of the map cannot be seen. By default <kbd>R</kbd> does that.
4. Try to find a good view that shows most of the map
5. Take a screenshot **from Minetest**. By default F12 does that.
6. You can find the screenshot in `screenshots` folder in the Minetest data folder unless you have changed the path in settings.
```

File: mods/ctf/ctf_map/README.md [hunk 2]
```diff
@@ -119,5 +119,11 @@ Areas where chests can be placed. Some maps allow placing chests anywhere, while
 ### 8. Screenshot
 
 If you choose to submit your map, include a screenshot of it in the exported map's folder. It should have an aspect ratio of 3:2 (screenshot 600x400px is suggested).
-
-It should be named `screenshot.png`.
+You can take a screenshot easily by doing the following:
+1. Hide the HUD. By default <kbd>F1</kbd> does that.
+2. Hide the chat log. By default <kbd>F2</kbd> does that.
+3. Try to find a good view that shows most of the map.
+4. *(Optional)* Enable unlimited view range if important parts of the map cannot be seen. By default <kbd>R</kbd> does that.
```
Comment (LoneWolfHT) [inline mods/ctf/ctf_map/README.md:126]:
```suggestion
4. *(Optional)* Increase your view range if important parts of the map cannot be seen. By default the <kbd>=</kbd> (or <kbd>+</kbd>) and <kbd>-</kbd> keys do that.
```

Comment (LoneWolfHT) [reply mods/ctf/ctf_map/README.md:126]: The `R` key is unbound in MT 5.7, and possibly earlier versions

**Pull request merged**